app/controllers/topics_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class TopicsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    render json: Topic.all
```

Comment:
This is fine, maybe for next prs you can add pagination, maybe this one? https://github.com/ddnexus/pagy

Reply:
Sure, maybe not for topics but for conversations or targets?
I believe topics are just 9, and the user cannot add/delete them

Reply:
Yeah, if that's the case don't bother to add pagination on topics

Reply:
https://trello.com/c/BZ7aXHLo/31-add-pagination-to-targets-index-action
https://trello.com/c/UkuF4YS0/32-add-pagination-to-conversations-index-action
Created these two.